tonic-reflection/Cargo.toml

```diff
@@ -21,13 +21,14 @@ version = "0.5.0"
 bytes = "1.0"
 prost = "0.11"
 prost-types = "0.11"
-tokio = {version = "1.0", features = ["sync"]}
+tokio = {version = "1.0", features = ["sync", "rt"]}
```

Comment:
Why does it need `rt`?

Reply:
> Why does it need `rt`?

It use `tokio::spawn` which available only with `rt` feature (one time pipeline was failed because of it).

Reply:
Hmm I wonder if we should move it to use `async-stream` instead but that can be for a future PR.